src/mobility/components/CarSharingStationBottomSheet.tsx

```diff
@@ -220,36 +215,30 @@ const useSheetStyle = StyleSheet.createThemeHook((theme) => {
       marginBottom: Math.max(bottom, theme.spacings.medium),
       marginHorizontal: theme.spacings.medium,
     },
+    icon: {
+      marginEnd: theme.spacings.xSmall,
+    },
     noCarsAvailable: {
       flex: 1,
       alignItems: 'center',
     },
     operatorButton: {
       marginTop: theme.spacings.medium,
     },
-    walkingDistance: {
-      marginBottom: theme.spacings.medium,
+    stationText: {
+      display: 'flex',
+      flexDirection: 'row',
+      marginTop: theme.spacings.xSmall,
     },
   };
 });
 
-const isAnyAvailable = (
+const totalAvailableCars = (
   vehicleTypesAvailable: CarAvailabilityFragment[] | undefined,
-) => {
-  const count =
+): number => {
```

Comment:
Just a tip since you are pretty new to Javascript, @reidzeibel. One liner functions like this can have implicit return. If you omit the brackets {} the function will return the value of your statement. This is equivalent to what you have:
```
const totalAvailableCars = (
  vehicleTypesAvailable: CarAvailabilityFragment[] | undefined,
) => vehicleTypesAvailable
      ?.map((v) => v.count)
      .reduce((sum, count) => sum + count, 0) ?? 0
```
You can also omit the explicit return type (`:number`). A good IDE (with the proper plugins) will deduce this and show it for you. 

![image](https://github.com/AtB-AS/mittatb-app/assets/4981580/b3a1d43f-7fe2-43b5-b1c0-59c292f73b1d)

Reply:
Thanks for the insight!